.github/workflows/pr-title-checker.yml

```diff
@@ -0,0 +1,49 @@
+name: PR Title
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  pr-title-check:
+    name: PR Title Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR title
+        uses: GetJobber/action-semantic-pull-request@v5
+        id: check_pr_title
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which scopes are allowed (newline delimited).
+          # These are regex patterns auto-wrapped in `^ $`.
+          scopes: |
```

Comment:
Purely curiosity, but what's the _intended_ outcome if a PR title includes multiple scopes? For example our recent typography branch which updated both design and components triggered a major bump to `3.0` as expected for `components`, but a [smaller bump](https://atlantis.getjobber.com/packages-design-changelog#0280-2023-02-07) for `design`. In my head I thought it would bump `design` to `1.0`.

Reply:
TBH, I'm not sure - that's a mechanism within lerna. I believe it looks at the git commits to try and figure things out though?